etc/env/example.env

```diff
@@ -197,3 +201,81 @@ DATASETS_SERVICE_PORT=8880
 AUTHORIZATION_SERVICE_VERSION=v0.0.1-alpha
 AUTHORIZATION_SERVICE_HOST=0.0.0.0
 AUTHORIZATION_SERVICE_PORT=7000
+
+
+# keycloak service
+CANDIG_AUTH_CONTAINER_NAME=candigauth.local
+KEYCLOAK_VERSION=9.0.2
+KC_ADMIN_USER=
+KC_ADMIN_PW=
+KC_TEST_USER=
+KC_TEST_PW=
+KC_TEST_USER_TWO=
+KC_TEST_PW_TWO=
+KC_REALM=candig
+KC_CLIENT_ID=cq_candig
+KC_LOGIN_REDIRECT_PATH=/auth/login
+## KC_SECRET= ###(generated in setup.sh)
+## KC_CLIENT_ID_64= ###(generated in setup.sh)
+## KC_PUBLIC_KEY= ###(generated in setup.sh)
+KEYCLOAK_SERVICE_PUBLIC_PORT=8081
+KEYCLOAK_SERVICE_CONTAINER_PORT=8081
+KEYCLOAK_SERVICE_HOST=0.0.0.0
+
+KEYCLAOK_SERVICE_PUBLIC_PROTO=http
+KEYCLAOK_SERVICE_PRIVATE_PROTO=http
+KEYCLOAK_SERVICE_PUBLIC_URL=${KEYCLAOK_SERVICE_PUBLIC_PROTO}://${CANDIG_AUTH_DOMAIN}:${KEYCLOAK_SERVICE_PUBLIC_PORT}
+KEYCLOAK_SERVICE_PRIVATE_URL=${KEYCLAOK_SERVICE_PRIVATE_PROTO}://${CANDIG_AUTH_CONTAINER_NAME}:${KEYCLOAK_SERVICE_CONTAINER_PORT}
+
+
+# tyk service
+CANDIG_SERVER_CONTAINER_NAME=candig_server_candig-server_1
+CANDIG_SERVER_PUBLIC_PORT=8080
+TYK_VERSION=v2.9.3.1
+TYK_REDIS_VERSION=4.0.14-alpine
+TYK_SERVICE_CONTAINER_NAME=tyk
+TYK_SERVICE_PUBLIC_PORT=8080
+TYK_SERVICE_HOST=0.0.0.0
+TYK_SERVICE_PRIVATE_URL=http://${TYK_SERVICE_CONTAINER_NAME}:${TYK_SERVICE_PUBLIC_PORT}
+TYK_TARGET_URL=http://${CANDIG_ARBITER_HOST}:${CANDIG_ARBITER_SERVICE_PORT}
+TYK_LOGIN_TARGET_URL=http://${TYK_SERVICE_HOST}:${TYK_SERVICE_PUBLIC_PORT}
+TYK_LISTEN_PATH=
+TYK_POLICY_ID=candig_policy
+
+## -- generic template stuff
+AUTH_API_ID=11
+AUTH_API_NAME=Authentication
+AUTH_API_SLUG=authentication
+
+CANDIG_API_ID=21
+CANDIG_API_NAME=CanDIG
+CANDIG_API_SLUG=candig
+CANDIG_PUBLIC_URL=http://${CANDIG_DOMAIN}:${CANDIG_SERVER_PUBLIC_PORT}
+## --
+
+
+# vault service
+VAULT_CONTAINER_NAME=vault
```

Comment:
could you reorganize this a bit, not really a rule but for consistency it should be service version, then service host ip, then service ports, then any additional values specific to the service